### PR TITLE
ci: fix path to the jar file

### PR DESCRIPTION
**Pull request by uladzimirdev**
### Description

Fix path to the jar file

~Attempt to bypass trivy rate limit problem~

example of failure https://github.com/metabase/metabase/actions/runs/11792759312/job/32847235666

~related issue https://github.com/aquasecurity/trivy-action/issues/389~

### CODE REVIEW

**Review by luizarakaki — APPROVED** [review]

**Review by luizarakaki — APPROVED** [review]
looks good

File: .github/actions/build-ee-extra/action.yml [hunk 1]
```diff
@@ -110,7 +110,7 @@ runs:
       shell: bash
     - name: Launch uberjar
       run: >-
-        java --add-opens java.base/java.nio=ALL-UNNAMED -jar ./metabase.jar &
+        java --add-opens java.base/java.nio=ALL-UNNAMED -jar ./target/uberjar/metabase.jar &
```
Comment (camsaul) [inline .github/actions/build-ee-extra/action.yml:113]: I think the old version was actually `./bin/docker/metabase.jar`

Comment (camsaul) [reply .github/actions/build-ee-extra/action.yml:113]: A bunch of other stuff too in `pre-release.yml`

Comment (camsaul) [reply .github/actions/build-ee-extra/action.yml:113]: Also in `pre-release.yml`

**Pull request merged**